src/components/ModalConfirm.vue

```diff
@@ -87,9 +87,9 @@ watch(
   >
     <div class="flex flex-auto flex-col">
       <h4 class="m-4 mb-0 text-center">
-        {{ $tc('proposal.castVote') }}
+        {{ $tc('voteOverview') }}
```

Comment:
@twodam why were these changes reverted?

Reply:
I believe it's because base branch are updated and you commit once before to manually `merge` them.
Anyway, check `Files changed` section, it should no longer touch these lines.

Reply:
I still don't see these changes I pushed to this PR. Can you add them back

Reply:
I would like this changes to be included that were removed here https://github.com/snapshot-labs/snapshot/commit/9ce757ac9505ce991377554b0e3b593e704b1422